indico/MaKaC/webinterface/tpls/Admins.tpl

```diff
@@ -86,10 +93,24 @@
 </table>
 
 <script>
+    $(function() {
+        var adminListManager = new ListOfUsersManager(null, {
+                'addExisting': 'admin.general.addExistingAdmin',
+                'remove'     : 'admin.general.removeAdmin'
+            }, {}, $E('inPlaceAdministrators'), "administrator", "item-user", false, {}, {
+                title      : false,
```

Comment:
I'm not sure if I like this alignment. We are certainly not using it in most parts of Indico.